### DISCUSSION TITLE
Which-key layout changed after latest update

### DISCUSSION THREAD

**Discussion by ukcw** [LazyVim/LazyVim#5159] [Q&A]
<img width="1712" alt="image" src="https://github.com/user-attachments/assets/93fba651-8fa8-4455-b92f-127283e43e1f" />

How can I get which-key popup to be in the center as before the recent update? Now which-key is showing up on the bottom right.

**Comment by dpetka2001** [comment]
```lua
return {
  {
    "folke/which-key.nvim",
    opts = {
      preset = "classic",
    },
  },
}
```

**Reply by ukcw** [reply]
thanks!
